tile_pages.py

```diff
@@ -214,15 +216,16 @@ def run(
                 page_names.append(pagekey)
 
                 if abs(pagembox[2] - refmbox[2]) > 1 or abs(pagembox[3] - refmbox[3]) > 1:
-                    print(_('Warning: page {} is a different size from {}, output may be unpredictable').format(p, ref_p))
-
+                    different_size.add(p)
+                
                 refmbox = pagembox
-                ref_p = p
```

Comment:
The reason I updated the reference page is because sewing patterns often have a bunch of instructions in one page size followed by pattern pieces in another, and then maybe instructions again, so I don't necessarily always want to compare to the first page. However, this is probably not what the user wants anyway, so maybe we could just do a general mixed page size warning without calling out specific pages.

I do want to continue updating the refmbox though, as I think it makes most sense for blank pages to be the size of the previous one (or the first non-zero, in the case where blank pages happen first).